### ISSUE TITLE
Syntax for $ as key

### ISSUE THREAD

**Issue by ycmjason** [othree/yajs.vim#174]
```js
fn({
  $a: 3
});
```

The `$` will not be highlighted properly:
![image](https://user-images.githubusercontent.com/1251825/50483298-26853100-09e3-11e9-9420-87bea876226b.png)

Expected:
![image](https://user-images.githubusercontent.com/1251825/50483308-3735a700-09e3-11e9-830a-3dec96367076.png)

Thanks for the good work!

**Comment by othree** [comment]
Hi, what is your Vim version?
I use `syntax iskeyword` to do this now.
It requires 7.4-1142

**Comment by ycmjason** [comment]
@othree Hi, I am currently using neovim. Perhaps the problem was with neovim?

**Comment by othree** [comment]
Sorry, I am not sure which version of nvim supports this feature.
Can you try execute manually:

```vim
syn iskeyword @,48-57,_,192-255,$
```

**Comment by ycmjason** [comment]
I have tried it, confirming that it will work!

**Comment by othree** [comment]
So I think nvim need other test method to see is it supports this feature.

**Comment by ycmjason** [comment]
@othree 

Yes. After running your suggested command: `syn iskeyword @,48-57,_,192-255,$` the highlight is correct.
